### PR TITLE
tuner: init at 1.5.1

### PR DESCRIPTION
**Pull request by wahjava**
###### Description of changes

https://github.com/louis77/tuner

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/tuner/default.nix [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+, desktop-file-utils
+, libsoup
+, json-glib
+, geoclue2
+, geocode-glib
+, libgee
+, gtk3
+, pantheon
+, gst_all_1
+, gst_plugins ? with gst_all_1; [ gst-plugins-good gst-plugins-ugly ]
```
Comment (AndersonTorres) [inline pkgs/applications/audio/tuner/default.nix:20]:
```suggestion
```

File: pkgs/applications/audio/tuner/default.nix [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    geoclue2
+    geocode-glib
+    libgee
+    glib
+    gtk3
+    pantheon.granite
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-bad
+  ] ++ gst_plugins;
```
Comment (AndersonTorres) [inline pkgs/applications/audio/tuner/default.nix:57]:
```suggestion
  ];
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/tuner/default.nix (hunk 1 above)
Comment (wahjava) [reply pkgs/applications/audio/tuner/default.nix:20]: Thanks, I followed `rhythmbox`. I'll include your suggestion, and re-push it shortly.

File: pkgs/applications/audio/tuner/default.nix [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+  ];
+
+  meta = with lib; {
+    homepage = "https://wiki.gnome.org/Apps/Rhythmbox";
+    description = "An app to discover and play internet radio stations";
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
```
Comment (zendo) [inline pkgs/applications/audio/tuner/default.nix:64]:
```suggestion
    platforms = platforms.linux;
    mainProgram = "com.github.louis77.tuner";
```

File: pkgs/applications/audio/tuner/default.nix [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    pantheon.granite
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+  ];
+
+  meta = with lib; {
+    homepage = "https://wiki.gnome.org/Apps/Rhythmbox";
```
Comment (zendo) [inline pkgs/applications/audio/tuner/default.nix:61]:
```suggestion
    homepage = "https://github.com/louis77/tuner";
```

Comment (wahjava) [reply pkgs/applications/audio/tuner/default.nix]: Thank you!

File: pkgs/applications/audio/tuner/default.nix (hunk 3 above)
Comment (wahjava) [reply pkgs/applications/audio/tuner/default.nix:64]: Thank you!

File: pkgs/top-level/all-packages.nix [hunk 5]
```diff
@@ -38208,4 +38208,6 @@ with pkgs;
   widevine-cdm = callPackage ../applications/networking/browsers/misc/widevine-cdm.nix { };
 
   alsa-scarlett-gui = callPackage ../applications/audio/alsa-scarlett-gui { };
+
+  tuner = callPackage ../applications/audio/tuner { };
```
Comment (AndersonTorres) [inline pkgs/top-level/all-packages.nix:38212]: Feels like the laziness is overflowing.

**Pull request merged**